bedrock/firefox/templates/firefox/features/pdf-complete-fr.html

```diff
@@ -0,0 +1,94 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "firefox/features/base-article.html" %}
+
+
+{% block article_title_short %}Remplir un PDF en ligne: fiche d’impôts{% endblock %}
+{% block article_title %}Remplir un PDF en ligne : fiche d’impôts{% endblock %}
```

Comment:
**Nit:** Based on the copy, these titles should be `Comment remplir sa fiche d’impôts en ligne via PDF? `?

Reply:
Adria/Leonie approved the copy changes :)